tasks/modules.py

```diff
@@ -117,6 +118,9 @@ def dependency_path(self, agent_version):
     "pkg/otlp/model": GoModule("pkg/otlp/model", independent=True),
     "pkg/security/secl": GoModule("pkg/security/secl", independent=True),
     "pkg/remoteconfig/state": GoModule("pkg/remoteconfig/state", independent=True),
+    "pkg/util/cgroups": GoModule("pkg/util/cgroups", condition=lambda: sys.platform == "linux"),
+    "pkg/util/log": GoModule("pkg/util/log"),
```

Comment:
According to `pkg/util/log/go.sum` and `pkg/util/cgroups/go.sum`, `pkg/util/log` and `pkg/util/cgroups` seem independent from the main `datadog-agent` module, why couldn't we add `independent=True` to them here?